src/components/common/Modal/index.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Button,
+  IconButton,
+  Modal as ModalContainer,
+  Text
+} from '@offer-ui/react'
+import type { ReactElement, ReactNode } from 'react'
+import Image from 'next/image'
+import styled from '@emotion/styled'
+import type { StyledProps } from '@types'
+
+export interface ModalProps {
+  isOpen?: boolean
+  hasLogo?: boolean
+  title: ReactNode
+  description?: string
+  primaryButtonText?: string
+  ghostButtonText?: string
+  children?: ReactNode
+  onClose(): void
+}
+type StyledLogoContainerProps = StyledProps<ModalProps, 'hasLogo'>
+
+export const Modal = ({
+  isOpen = false,
+  onClose,
+  hasLogo = false,
+  title,
+  description = '',
+  primaryButtonText = '',
+  ghostButtonText = '',
+  children
+}: ModalProps): ReactElement => {
+  return (
+    <StyledModalContainer isOpen={isOpen} onClose={onClose}>
+      <StyledHeader>
+        {hasLogo && (
+          <StyledLogoContainer hasLogo={hasLogo}>
```

Comment:
```suggestion
          <StyledLogoWrapper hasLogo={hasLogo}>
```
요기 Wrapper 의도하신게 맞을까요!?

Reply:
수정 완료 🤩 (43ebb4b73f2863b34bc6a71204e76e97c1b6dccc)
불필요해서 제거했십니더